detect_secrets/plugins/slack.py

```diff
@@ -10,8 +10,16 @@
 
 class SlackDetector(RegexBasedDetector):
 
-    secret_type = 'Slack Token'
+    secret_type = 'Slack Token / Webhook'
```

Comment:
This is going to be a [non-backwards compatible change](https://github.com/Yelp/detect-secrets/blob/master/CHANGELOG.md#boom-breaking-changes-3).

Are you sure you want to do this?

Reply:
(Backwards compatibility aside)

I think we may want to make a different plugin, I say this because we should only have 1 `verify` method once #194 is merged, and verifying token's vs. webhooks is different.

Reply:
I'm fine with having the same plugin.

In the `verify` method, we can do:

```
def verify(self, token):
    if token.startswith('https://hooks.slack.com'):
        # do logic
```

Reply:
As far as backwards compatibility, I don't mind but there may be a user that does. Slack isn't one of the noisier plugins so I'd say this is okay :) What do you think?

(To explain better, the reason this breaks backwards compatibility is because all of a user's old acknowledged Slack tokens will be re-alerted on, because the "[hashed_secret](https://github.com/Yelp/detect-secrets/blob/master/CONTRIBUTING.md#technical-details)" value will change.)

Reply:
I'm in favor of the slightly less descriptive, but more compatible decision to keep `secret_type`. IMO, adding the clarity that it also catches webhooks is less important.

Reply:
I would agree with @domanchi on more compatible approach.

Reply:
No objection here. I'll leave the type as it was.